generators/server/templates/gradle/_liquibase.gradle

```diff
@@ -63,12 +70,18 @@ task liquibaseDiffChangelog(dependsOn: compileJava, type: JavaExec) {
     args "--password="
     args "--url=<% if (devDatabaseType === 'mysql') { %>jdbc:mysql://localhost:3306/<%= baseName %><% } else if (devDatabaseType === 'mariadb') { %>jdbc:mariadb://localhost:3306/<%= baseName %><% } else if (devDatabaseType === 'postgresql') { %>jdbc:postgresql://localhost:5432/<%= baseName %><% } else if (devDatabaseType === 'h2Disk') { %>jdbc:h2:file:./<%= BUILD_DIR %>h2db/db/<%= lowercaseBaseName %><% } else if (devDatabaseType === 'oracle') { %>jdbc:oracle:thin:@localhost:1521:<%= baseName %><% } %>"
     args "--driver=<% if (devDatabaseType === 'mysql') { %>com.mysql.jdbc.Driver<% } else if (devDatabaseType === 'mariadb') { %>org.mariadb.jdbc.Driver<% } else if (devDatabaseType === 'postgresql') { %>org.postgresql.Driver<% } else if (devDatabaseType === 'h2Disk') { %>org.h2.Driver<% } else if (devDatabaseType === 'oracle') { %>oracle.jdbc.OracleDriver<% } %>"
-    args "diffChangeLog"
+    args liquibaseCommand
     <%_ if (authenticationType === 'oauth2') { _%>
     args "--excludeObjects=oauth_access_token, oauth_approvals, oauth_client_details, oauth_client_token, oauth_code, oauth_refresh_token"
     <%_ } _%>
 }
 
+// for compatibility to former task
+task liquibaseClearChecksums() {
```

Comment:
Shouldn't it be named `liquibaseDiffChangelog`?

Reply:
Ups, sure, sorry! Must have been some copy-paste error - thanks for reviewing!